wagtaildraftail/client/wagtaildraftail.js

```diff
@@ -35,9 +45,9 @@ const initDraftailEditor = (fieldName, options = {}) => {
     // eslint-disable-next-line no-param-reassign
     options.entityTypes = options.entityTypes.map(entity => Object.assign(entity, {
       // TODO: Rename keys accordingly once changed in draftail https://github.com/springload/draftail/issues/49
-      control: sources[entity.source],
-      strategy: getEntityStrategy(entity.type),
-      component: decorators[entity.decorator],
+      control: getSource(entity.source),
+      strategy: getStrategy(entity.type) || getDefaultStrategy(entity.type),
+      component: getDecorator(entity.decorator),
```

Comment:
I think the `wagtaildraftail` object should be defined above, so this code can use the same `wagtaildraftail.getSource` and so on as the userland code.